src/pages/academy/components/_search-banner.tsx

```diff
@@ -377,7 +378,18 @@ const SearchBar = ({ setModal, setHideMobileTopic }: SearchBarProps) => {
 
     const input_ref = useRef<HTMLInputElement>()
 
+    if (is_eu && !is_uk) {
+        academy_data.blog = academy_data.blog.filter(
+            (item) => item.visibility !== 'hide_for_eu' && item.visibility !== 'hide_for_eu_uk',
+        )
+    } else if (is_uk) {
+        academy_data.blog = academy_data.blog.filter(
+            (item) => item.visibility !== 'hide_for_uk' && item.visibility !== 'hide_for_eu_uk',
+        )
+    }
+
```

Comment:
Since we have used the updated helper for `eu_country` no need for this part

```
&& item.visibility !== 'hide_for_eu_uk',
```

Also 

```
 if(is_eu)
 ```
 
 is enough

Reply:
Agree with second state. But with first...
With `&& item.visibility !== 'hide_for_eu_uk'` = 79 topics
Without = 85
![Screenshot 2022-05-17 at 13 16 40](https://user-images.githubusercontent.com/103182473/168789110-4cef0ee0-c223-45b1-b0dd-56a929ca3193.png)
![Screenshot 2022-05-17 at 13 16 26](https://user-images.githubusercontent.com/103182473/168789115-74691afc-0cf4-427d-aec9-641057f67610.png)

Reply:
It seems to me that data don't filter with helper you wrote about. In search banner data fetch using this `const { academy_data } = useContext(DerivStore)`